src/index.ts

```diff
@@ -0,0 +1,55 @@
+import { app, startServer } from "./libs/rest";
+import { setRoutes } from "./routes/index";
+import { handleErrors } from "./utils/errors";
+import { downloadLatestSignsData } from "./services/zodiacdeep";
+
+//// zodiac?sign=leo
+//// http://localhost:3000/lucky?luckyText=lucky
+// www -> http://localhost:3000/lucky?lucky=lucky
+
+(async function (): Promise<void> {
+  try {
+    startServer();
+    setRoutes(app);
+    // Save today' all signs data into DB
+    // dailyDownloadLatestSignsData();
+    // dailyDownloadLatestSignsDataFORTESTS();
+  } catch (error) {
+    console.log(error);
+  }
+})();
+
+export async function dailyDownloadLatestSignsData(): Promise<void> {
```

Comment:
The place of this function is not in the main index. file. From what I see - this is business logic.